public/sitemap.xml

```diff
@@ -0,0 +1,64 @@
+<!-- public/sitemap.xml -->
```

Comment:
@RahatH07 looks like sonar has an issue with this line too, I know it's only a comment but can you remove too please?